CHANGES.rst

```diff
@@ -48,6 +48,7 @@ Improvements
 - Allow importing event role members from a CSV file (:issue:`4301`)
 - Allow optional comment when accepting a pre-booking (:issue:`4086`)
 - Log event restores in event log (:issue:`4309`)
+- Enhance cancellation of recurring bookings (:issue:`4092`)
```

Comment:
I'd go a bit more into detail here. What about something like this?

```rst
- Warn about cancelling/rejecting whole recurring bookings instead of just
  specific occurrences (:issue:`4092`)
```